init.d/gdrcopy

```diff
@@ -78,6 +91,12 @@ load_module()
 # Start daemon
 function start() {
 
+    echo -n $"Checking required modules: "
```

Comment:
@spotluri with this change, the init script will check for the presence of the nvidia driver